src/block/Block.php

```diff
@@ -454,11 +455,21 @@ public function onBreak(Item $item, ?Player $player = null, array &$returnedItem
 
 	/**
 	 * Called when this block or a block immediately adjacent to it changes state.
+	 *
+	 * @deprecated {@see Block::onNearbyBlockChange2()}
```

Comment:
I'm not sure if this actually works. Better to put the @see on its own line rather than writing it inline.